api/routes/cities.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
```

Comment:
```json
{
country: "AD",
count: 65220,
locations: 2
}
```
Looks like we're missing `city` and `name` if you want to move people towards that as well.